src/webhook/mutation/pod_mutator/oneagent_mutation/config.go

```diff
@@ -0,0 +1,33 @@
+package oneagent_mutation
+
+import (
+	"github.com/Dynatrace/dynatrace-operator/src/logger"
+)
+
+var (
+	log = logger.NewDTLogger().WithName("pod.mutation-webhook.oneagent")
+)
+
+const (
+	preloadEnvVarName           = "LD_PRELOAD"
+	networkZoneEnvVarName       = "DT_NETWORK_ZONE"
+	proxyEnvVarName             = "DT_PROXY"
+	dynatraceMetadataEnvVarName = "DT_DEPLOYMENT_METADATA"
+
+	oneAgentBinVolumeName   = "oneagent-bin"
+	oneAgentShareVolumeName = "oneagent-share"
+
+	injectionConfigVolumeName = "injection-config"
+
+	provisionedVolumeMode = "provisioned"
+	installerVolumeMode   = "installer"
+
+	oneAgentCustomKeysPath = "/var/lib/dynatrace/oneagent/agent/customkeys"
+	customCertFileName     = "custom.pem"
+	proxySecretKey         = "proxy"
```

Comment:
nit: could reuse const from `properties` package so we don't redefine the proxy key